core/definitions/src/testing/mocks/tokenBridge.ts

```diff
@@ -47,16 +44,16 @@ export class MockTokenBridge<P extends PlatformName> implements TokenBridge<P> {
     throw new Error("Method not implemented.");
   }
   transfer(
-    sender: UniversalOrNative<P>,
+    sender: AnyAddress,
     recipient: ChainAddress,
-    token: "native" | UniversalOrNative<P>,
+    token: "native" | AnyAddress,
```

Comment:
AnyAddress has `native`  already, can we get rid of this union type?

Reply:
string is already supported, so I just removed 'native'